### PR TITLE
a little bit more documentation

### PR DESCRIPTION
**Pull request by mjoe999**

### CODE REVIEW

**Review by Cj-Malone — APPROVED** [review]

File: docs/WEEKLY_RUN.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+that they can be run regularly. This is important as things in the real world
+change regularly. Companies open new and close old locations. Opening hours can
+change. Also, a company can change their website in a way that causes the spider
+to no longer function.
+
+### alltheplaces.xyz
+
+For all these reasons we try (at the time of writing) to perform a weekly run of
+all the spiders. The output is published on our
+website: [alltheplace.xyz](https://www.alltheplaces.xyz/).
```
Comment (iandees) [inline docs/WEEKLY_RUN.md:13]:
```suggestion
website: [alltheplaces.xyz](https://www.alltheplaces.xyz/).
```

File: docs/WEEKLY_RUN.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Given that we publish the data, there should be no reason for you to run all the
+spiders yourself. If we all started doing this then a site is far more likely to
+engage blocking measures.
+
+### Statistics
+
+Alongside the POI data we further publish reports that help us maintain the
+health of the code base. For a list of available reports then see the links on
+[alltheplace.xyz](https://www.alltheplaces.xyz/). Some examples of the
```
Comment (iandees) [inline docs/WEEKLY_RUN.md:27]:
```suggestion
[alltheplaces.xyz](https://www.alltheplaces.xyz/). Some examples of the
```

**Review by iandees — APPROVED** [review]

**Comment by mjoe999** [comment]
ah @iandees! I have just seen your edits to DATA_FORMAT.md .... I live in my own little world! :-)

**Comment by iandees** [comment]
Not a problem! A bit of repetition is good when it comes to this stuff.

**Pull request merged**